pyproject.toml

```diff
@@ -58,12 +58,72 @@ Documentation = "https://pymediainfo.readthedocs.io/"
 Bugs = "https://github.com/sbraz/pymediainfo/issues"
 
 
+# Custom entries for downloading the bundled libmediainfo
+[tool.bundled_libmediainfo]
+version = "24.12"
```

Comment:
Nice, I like the idea of having the variable stuff in a config file instead of a script :D